ceph_deploy/osd.py

```diff
@@ -322,8 +328,11 @@ def prepare(args, cfg, activate_prepared_disk):
                       hostname, disk, journal, activate_prepared_disk)
 
             storetype = None
+            block = None
             if args.bluestore:
                 storetype = 'bluestore'
+                block = journal
```

Comment:
@zenglg I think you want to explain bluestore expects BLOCK (as compared to JOURNAL in filestore) and append that coment before this so as to avoid confusion.